contracts/src/v0.8/shared/test/helpers/ChainReaderTester.sol

```diff
@@ -8,6 +8,7 @@ struct TestStruct {
   uint8 OracleId;
   uint8[32] OracleIds;
   address Account;
+  address AccountStr;
```

Comment:
You don't have to make any changes to the tester contract, you can just change the type `TestStruct struct {}
` in common to have the Account fields as a string